doc/v2/documentation_data_format.rst

```diff
@@ -1097,6 +1096,52 @@ Conditions and observables that are not applied to a model do not need to be
 valid for that model.
 
 
+.. _v2_initialization_semantics:
+
+Initialization and parameter application
+----------------------------------------
+
+The following describes how the model is initialized and how other PEtab
+parameter values are applied before the simulation starts.
+This procedure is executed for each model and experiment combination
+defined in the PEtab problem.
+
+1. Pre-initialization
+
+   1. Parameters values for parameters that occur in the parameter table are
+      applied to the uninitialized model.
```

Comment:
Clarify what uninitialized means here?

Reply:
This will probably be model-format-dependent. I can add "pre-initial-assignment-execution" as an example for SBML models. Keeping SBML specifics out of the PEtab specs didn't work too well anyways... :see_no_evil: